test/integration_test.rb

```diff
@@ -0,0 +1,101 @@
+require 'test_helper'
+require 'json'
+require 'root/root'
+require 'root/root_v2_api'
+require 'smart_proxy_remote_execution_ssh/plugin'
+
+class SmartProxyRemoteExecutionSshApiFeaturesTest < MiniTest::Test
```

Comment:
There is currently no test for the legacy async-ssh mode. Would probably be useful.